Dockerfile

```diff
@@ -21,6 +21,11 @@ COPY Pipfile* /tmp/
 RUN cd /tmp \
     && pipenv install --system
 
+RUN mkdir -p /config
```

Comment:
we're using Pipenv to manage dependencies in this project at the moment - have a look at the README. I can help you get things working with Pipenv if you'd like?